solr-updater/src/main/java/dk/dbc/search/solrdocstore/updater/profile/Profile.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2019 DBC A/S (http://dbc.dk/)
+ *
+ * This is part of solr-doc-store-updater
+ *
+ * solr-doc-store-updater is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * solr-doc-store-updater is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package dk.dbc.search.solrdocstore.updater.profile;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ *
+ * @author Morten Bøgeskov (mb@dbc.dk)
+ */
+public class Profile implements Serializable {
```

Comment:
Kunne denne klasse med fordel ligge i en maven package som man kunne have en dependency på? Skal profiles ikke bruges af andre? f.eks. profile service? Specielt hvis profile service skal ligge uden for teamet på sigt, så må det med profiles være rimelig generelt.

Reply:
Denne klasse er til at parse profile-service (vores) modellen. Jeg er blevet belært i forbindelse med skift fra rawrepo-content- til rawrepo-record-service, at vi ikke vil have dependencies på DTO'er (data transfer object). At vi i stedet skal have dem integreret i projektet og kunne håndtere tilføjelser i dem.
Derudover har jeg ikke lyst til at presse en datamodel ned over at andet team (java-openagency), som formodentligt vil have en bedre (mere generel) struktur som ligger tættere på databasen.